### PR TITLE
Chore Identifiers Cleanup

### PR DESCRIPTION
**Pull request by ThunderSon**
**What did this PR accomplish?**
- Rename the group identifiers to follow a 4 letter convention:
1. WSTG-INFO-01
1. WSTG-CONF-01
1. WSTG-IDNT-01
1. WSTG-ATHN-01
1. WSTG-ATHZ-01
1. WSTG-SESS-01
1. WSTG-INPV-01
1. WSTG-ERRH-01
1. WSTG-CRYP-01
1. WSTG-BUSL-01
1. WSTG-CLNT-01
- Remove the 3rd `0` in the identifiers
- Update the whole guide accordingly

### CODE REVIEW

**Comment by kingthorin** [comment]
> FILE:document/4-Web_Application_Security_Testing/04-Authentication_Testing/02-Testing_for_Default_Credentials.md
[✖] ../4.5_Authentication_Testing/07-Testing_for_Weak_Password_Policy.md → Status: 400

File: document/4-Web_Application_Security_Testing/09-Testing_for_Weak_Cryptography/01-Testing_for_Weak_SSL_TLS_Ciphers_Insufficient_Transport_Layer_Protection.md [hunk 1]
```diff
@@ -988,7 +988,7 @@ Checking localhost:443 for ROBUST use of anti-caching mechanism ...
 Robust Solution:
 
     - Cache-Control: no-cache, no-store, must-revalidate, pre-check=0, post-check=0, max-age=0, s-maxage=0
-    - Ref: https://www.owasp.org/index.php/Testing_for_Browser_cache_weakness_(WSTG-AUTHN-006)
+    - Ref: https://wiki.owasp.org/index.php/Testing_for_Browser_cache_weakness_(OTG-AUTHN-006)
```
Comment (kingthorin) [inline document/4-Web_Application_Security_Testing/09-Testing_for_Weak_Cryptography/01-Testing_for_Weak_SSL_TLS_Ciphers_Insufficient_Transport_Layer_Protection.md]: Why not reference the local (internal) material?

Comment (ThunderSon) [reply document/4-Web_Application_Security_Testing/09-Testing_for_Weak_Cryptography/01-Testing_for_Weak_SSL_TLS_Ciphers_Insufficient_Transport_Layer_Protection.md]: That whole section needs re-writing. I don't want to reference something new in it. I'd just flip and do it all by mistake 🙄 Take this in as it is or you're receiving another PR 😋

Comment (kingthorin) [reply document/4-Web_Application_Security_Testing/09-Testing_for_Weak_Cryptography/01-Testing_for_Weak_SSL_TLS_Ciphers_Insufficient_Transport_Layer_Protection.md]: K

**Comment by kingthorin** [comment]
>  FILE:document/4-Web_Application_Security_Testing/10-Business_Logic_Testing/06-Testing_for_the_Circumvention_of_Work_Flows.md
[✖] https://owasp.org/www-project-cheat-sheets/cheatsheets/Abuse_Case_Cheat_Sheet.html → Status: 404

https://cheatsheetseries.owasp.org/cheatsheets/Abuse_Case_Cheat_Sheet.html

**Comment by kingthorin** [comment]
Current link check failure seems to have been a temporary 500

**Comment by kingthorin** [comment]
It doesn't have to be this PR but it looks like the checklist and reporting section need updates. For example the checklist has 7 IDNT entries but the guide has only 5.

**Pull request merged**